### DISCUSSION TITLE
Anyone have luck with cheap Chinese Audio Interfaces (Depusheng U6)?

### DISCUSSION THREAD

**Discussion by wirelessguyny** [jamulussoftware/jamulus#1898] [Technical Support & Help]
Hi there.

I have an older Lexicon Omega that seems to work nicely with Jamulus and my Windows 10 PC but it likes to overheat.

I recently went to Amazon and bought a 6 channel mixer under the brand name Depusheng U6. It allows for connection via USB to the PC and shows up as JieLi BR17 in my windows interface. I'm using it together with ASIO4ALL v2.15 Beta 1

My guess is that it's the unit at fault and not ASIO but here's what I'm getting.

There is a button above the 5/6USB channel that when depressed allows me to hear the others playing on a Jamulus stream. When I press the button it switches from USB to ST mode and I hear nothing. Having said this, when I DO hear the stream it also seems to loop back and tries to play that out of the mixer, causing endless feedback.

Basically I'm stuck. Either I hear no one and they can likely hear me or I hear them and have to stay on mute in order to avoid utter mayhem. Has anyone run into this?

**Comment by ann0see** [comment]
> seems to loop back and tries to play that out of the mixer, causing endless feedback.

Does this mean you hear yourself twice (= direct monitoring: https://jamulus.io/wiki/Client-Troubleshooting [see "You all sound OK, but it’s difficult to keep together"]) or do you hear actual feedback (loud squeal)?

Depending on what happens, maybe you need to disable local monitoring —> see manual of mixer or use other in/outputs which are not connected. 

Since you’re using ASIO4ALL which is not perfect since it’s an universal driver I‘d suggest to search for an ASIO driver from the manufacturer. This might have more options. 

You could also take a photo from the device. Maybe some skilled audio engineer understands how it works?
